src/providers/ThemeProviderV2.tsx

```diff
@@ -0,0 +1,91 @@
+'use client';
+
+import * as React from 'react';
```

Comment:
we should rename this file to have the consistency between `theme` and `themeMode`